erpnext/setup/setup_wizard/utils.py

```diff
@@ -1,15 +0,0 @@
-import json, os
```

Comment:
don't delete this! I use it all the time for testing 😜

Reply:
okay :)